js/src/ui/AccountCard/accountCard.js

```diff
@@ -58,28 +58,45 @@ export default class AccountCard extends Component {
         onFocus={ this.onFocus }
         onKeyDown={ this.handleKeyDown }
       >
-        <div className={ styles.infoContainer }>
-          <IdentityIcon address={ address } />
-          <div className={ styles.accountInfo }>
-            <div className={ styles.accountName }>
-              <IdentityName
-                address={ address }
-                name={ name }
-                unknown
-              />
+        <div className={ styles.mainContainer }>
```

Comment:
While mucking about - we should really use a Container for these, matches up with what we have everywhere else.